### DISCUSSION TITLE
Welcome to telegram-bot-dialogs Discussions!

### DISCUSSION THREAD

**Discussion by alies-dev** [koot-labs/telegram-bot-dialogs#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Naro8686** [comment]
Good afternoon everyone, I'm interested in how to get the current dialog in the command and change step ?

**Reply by lptn** [reply]
Hey @Naro8686 
1. `DialogManager` has `getDialogInstance` method, but it's private so far. Feel free to create a PR to make it public (but please share arguments)
2. "change step" - I'm not sure it's a good idea. Most likely you are doing something wrong if you need to do it. It's like `goto` in programming - consequences are unpredictable :)

**Comment by dimaodnokoz** [comment]
Hello, @alies-dev. Is it ok if i copy (not fork) your telegram-bot-dialogs, develop and maintain it independantly on my own? The reason is i'm new here in github and trying to make my start as simple as possible. I'd like to feel protected from my unexpected sync errors or something. The other reason is process of merge and PRs - it is not clear for me now. I need a lot of time to understand other things and these is like more time. Of cource i will mention you and your package in my description and keep it there. Or please advice me something better here.

**Reply by dimaodnokoz** [reply]
@alies-dev, i saw your reply to my same question in closed discussion about bug, where i wrote it first. As you replied, i may do as i described, the only thing - i need to follow MIT license guidelines. Yes, of course. Thank you. As you also mentioned, you will maintain this package, so my conclusion was wrong. Sorry for that.
